tools/custom/raw-bytecode-analyser/README.md

```diff
@@ -0,0 +1,92 @@
+# Hedera Smart Contract Bytecode Analyser
+
+This Python script analyses the bytecode of Smart Contracts on the Hedera network to detect specific operand usages.
+It can help determine the likelihood that a Smart Contract creates a token using an SECP256k1 key.
+
+## Requirements
+
+- Python 3.6 or higher
+- `requests` library
+- `requests-cache` library: A transparent, persistent cache for the `requests` library to improve performance by caching HTTP responses.
+- `evmdasm` library: A tool for disassembling EVM bytecode to human-readable assembly instructions.
+
+To install the required Python libraries, run:
+
+```bash
+pip install requests requests-cache evmdasm
+```
+
+## Usage
+
+The script can be run from the command line with the following arguments:
+
+- `contract_id`: The ID of the smart contract whose bytecode you want to analyze.
+- `--previewnet`: Optional flag to use the Previewnet Mirror Node URL.
+- `--testnet`: Optional flag to use the Testnet Mirror Node URL.
+
+If no network flag is specified, the script defaults to using the Mainnet Mirror Node URL.
```

Comment:
I would prefer we switch this. Default to testnet and let people set a flag to point to mainnet.